docs/src/development/ssh.md

```diff
@@ -2,131 +2,118 @@
 title: "Using SSH"
 weight: 12
 description: |
-  One of the ways Platform.sh keeps things secure is by using SSH behind the scenes. Users can interact with their environment through a command shell, or push changes to the environment's Git repository, and both of these features rely on SSH.
+  Platform.sh uses Secure Shell (SSH) to help keep things secure. You can interact with your deployed environments or use the Platform.sh CLI, and both of these features rely on SSH.
 sidebarTitle: "SSH"
 ---
 
 {{< description >}}
 
-Secure Shell Protocol, SSH, supports certificate-based and keypair-based authentication.  Certificate-based authentication is faster to set up and generally easier to use, provided you have a web browser available on your computer.  Alternatively, you may use keypair-based authentication if you are setting up an automation tool, or simply prefer that method.
+This means you can do things like securely log in to your deployed server to troubleshoot and see what's different than your local environment or read logs. And push changes to your Git repository. All secured through SSH.
 
-Automation tools may also use an [API Token](/development/cli/api-tokens.md).
+SSH supports authentication with certificates or key pairs. Certificates are faster to set up and generally easier to use, provided you have a web browser available. They are also required for the Platform.sh CLI (unless using an [API token](/development/cli/api-tokens.md)) and when your organization has multifactor authentication set up.
 
-## Certificate-based authentication
+You can use key pairs to access deployed environments if you are setting up an automation tool or prefer that method.
 
-To connect using certificate-based authentication, install the [Platform.sh CLI](/development/cli/_index.md).
+Automation tools may also use [API tokens](/development/cli/api-tokens.md).
 
-Once installed, you may run `platform login` or any CLI command that would require authentication.  In either case, a browser window will open and ask you to login with your Platform.sh account credentials.  This web page is already encrypted with TLS over HTTP, making it secure.
+## Authenticate with certificates
```

Comment:
```suggestion
## Authenticate with the Platform CLI
```

I think I see these as rather CLI which uses certificates underneath vs SSH Keys, rather than certs vs key pairs @CollierCZ what you do think? this would mean a different set of changes across this page 🤔

Reply:
I understand what you're saying. From a user perspective, they shouldn't have to really care if it's certificates or what underneath, just that they log in through the CLI. So I could see making that change.

But then I feel like the article isn't about SSH and its entire structure and placement need to be rethought. Right now, it starts as explaining the technology behind the connection to increase the feeling of security. That's why there are the two methods described as they are. If the focus is on user tasks, I'd rename it somehow and rethink it more.

I guess it's a question of the orientation of the docs also in the future. So what do you think of that @MariadeAnton?

Reply:
Yes, will rewrite it.